### PR TITLE
Add namespace support to the `replace-static-breakpoint-mixins` migrations

### PR DESCRIPTION
**Pull request by aaronccasanova**

### CODE REVIEW

File: polaris-migrator/README.md [hunk 1]
```diff
@@ -72,6 +72,19 @@ Replace the legacy Sass `spacing()` function with the supported CSS custom prope
 npx @shopify/polaris-migrator replace-sass-spacing <path>
 ```
 
+### `replace-static-breakpoint-mixins`
+
+Replace legacy static breakpoint mixins with the new Polaris tokens [media query variables](https://github.com/Shopify/polaris/blob/main/documentation/guides/migrating-from-v9-to-v10.md#media-query-variables).
```
Comment (aaronccasanova) [inline polaris-migrator/README.md]:
```suggestion
Replace legacy static breakpoint mixins with the new Polaris [media query variables](https://github.com/Shopify/polaris/blob/main/documentation/guides/migrating-from-v9-to-v10.md#media-query-variables).
```

File: polaris-migrator/src/utilities/sass.ts [hunk 2]
```diff
@@ -1,5 +1,18 @@
 import type {Node, ParsedValue, FunctionNode} from 'postcss-value-parser';
 
+/**
+ * Non-exhaustive CSS ident token RegExp:
+ * - https://drafts.csswg.org/css-syntax-3/#ident-token-diagram
+ * - https://www.w3.org/TR/CSS21/syndata.html#:~:text=%5B%2D%5D%3F%7Bnmstart%7D%7Bnmchar%7D*
+ */
+export const identTokenRegExp = /-?[_a-zA-Z][_a-zA-Z0-9-]*/;
```
Comment (sam-b-rose) [inline polaris-migrator/src/utilities/sass.ts]: Nice RegExp utilities! Also nice reference links.

For my own understanding though, can you clarify or give an example of why this is needed/helpful for the namespace?

**Review by sam-b-rose — COMMENTED** [review]
Nice! Just want to understand how and why to use the RegExp functions. Seem fancy, but I'm just curious

**Review by lgriffee — APPROVED** [review]

File: polaris-migrator/src/utilities/sass.ts (hunk 2 above)
Comment (aaronccasanova) [reply polaris-migrator/src/utilities/sass.ts]: Update: Discussed offline and removed. While it was valuable to understand how CSS/Sass identifiers are constructed, from a specification perspective, reimplementing them with RegExps adds little value to our migrations and could cause confusion for newcomers to the platform.

**Pull request merged**